packages/ui/src/Terminal.tsx

```diff
@@ -92,9 +92,9 @@ export const Terminal: FC<TerminalProps> = memo(
 
         addExecutionHistoryItem({ command: inputVal, res, prompt })
       } catch (err) {
-        // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-        const errorTyped: Error = err
-        addExecutionHistoryItem({ command: inputVal, error: errorTyped.message, prompt })
+        if (err instanceof Error) {
```

Comment:
What other types of `err` could it be? Is it OK that we just ignore them?

Reply:
It's the base type of Error. I think we catch them all in this situation. And have a certain type for needed data